Cargo.toml

```diff
@@ -4,7 +4,10 @@ members = ["cli", "lib"]
 resolver = "2"
 
 [workspace.package]
-rust-version = "1.65"
+rust-version = "1.71"
 
 [profile.release]
-strip = true
+strip = true      # Automatically strip symbols from the binary.
+lto = true        # Link-time optimization.
+opt-level = "s"   # Optimize for speed.
+codegen-units = 1 # Maximum size reduction optimizations.
```

Comment:
Do this things produce a visible / comparable impact?

Reply:
For me the binary release goes from 14mb to 12mb